File: admin-frontend/travalen-ui-admin-master/src/app/data-access/services/finance.service.ts
import { Injectable } from '@angular/core';
import { CommonService } from './common.service';
import { FinanceModel } from '../model/finance.model';

@Injectable({
  providedIn: 'root'
})
export class FinanceService {

  constructor(private commonService: CommonService) { }

  saveFinance(req: FinanceModel) {
    return this.commonService.httpPOST("finance/save",req);
  }

  getFinance(req: any) {
    return this.commonService.httpPOST("finance",req);
  }
}

File: New_admin-backend/travelo-admin/src/main/java/com/app/travelo/services/FinanceService.java
package com.app.travelo.services;

import com.app.travelo.model.rest.FinanceDto;
import com.app.travelo.model.rest.ResponseDto;

public interface FinanceService {
    ResponseDto<FinanceDto> saveFinance(FinanceDto req);
    ResponseDto<FinanceDto> getFinance(FinanceDto req);
}
